test/expressions/new-expr/new-on-domain.good

```diff
@@ -0,0 +1,2 @@
+new-on-domain.chpl:1: In function 'main':
+new-on-domain.chpl:2: error: invalid use of 'new' on chpl__buildDomainRuntimeType
```

Comment:
Is there a prettify function you could use on this error message so that we don't bleed internal compiler implementation details?